src/extension/src/Constants.py

```diff
@@ -27,6 +27,9 @@ def __iter__(self):
                     if item == self.__dict__[item]:
                         yield item
 
+    # Extension version (todo: move to a different file)
+    EXT_VERSION = "1.6.12"
+
```

Comment:
Needs to be addressed in full PR.

Reply:
To be sure, thi is teh part about moving this Constant to a different file right?

Reply:
Make it get populated as part of build/compilation from a different file that may write it across any source that needs it. Just as a note for full PR, may be resolved here. Non-issue as this whole PR needs to be reverted after validation.